### ISSUE TITLE
Balance control disappeared

### ISSUE THREAD

**Issue by kiwerry** [jfchapman/VUPlayer#27]
Thanks very much, James - VUplayer 2.49 has been in my startup folder for quite a few years now. 
Did I miss something, or has the Balance slider gone in V. 4.13? Likely to come back?
Background
I googled VUPlayer today because I was on the lookout for a portable version, and was delighted to find 4.13 available for download. On starting it with the -portable key it worked as expected, but I couldn't find a Balance control, which I need because my hearing is very lopsided as the result of an infection on one side about 60 years ago...

**Comment by jfchapman** [comment]
Hi,
The balance control was one of the features that never got copied over to the new version, but it will be in the next release.

Thanks,
James

**Comment by kiwerry** [comment]
Thanks, James, looking forward to that.

**Comment by jfchapman** [comment]
The latest release (v4.14) now has a balance control (selectable from the main 'View' menu, or from the slider control context menu). There's also an option (in the general options page) to retain the pitch & balance levels (by default they are reset when the application restarts).

Thanks,
James

**Comment by kiwerry** [comment]
Thanks very much James.

As you are still working on VUPlayer I thought that the following strange behaviour which occurred in the latest version occurred when I tried it out may be of interest to you.

 

Installation from 64 bit zip file, started using the portable option described in your readme:

Every time the program was started it had a blank playlist. All other options (balance setting, background colour, artwork jpg file ) were correct. I opened the one of my playlists (it opened at the correct path), and the playlist opened and the session ran normally, except for the black artwork problem described below. If I closed the program and reopened it. It had once again failed to read the playlist. I tried the 32 bit zip file installation from zip as above, same result. I then downloaded the setup.exe and installed the program conventionally, which fixed the playlist file not being read problem, but not for a portable installation. 

 

A further interesting problem which occurred in all three of the above installations was that the artwork box window turned black depending on the song selected. Some songs displayed an album cover, some my Artwork jpg, and some blackness; In one case I noticed two versions of the same song, stored in different folder, exhibited this behaviour – select one, see artwork; select the other, see blackness. 

A check of the Artwork in the setup revealed that the correct file was still selected, even displayed in the Artwork setup window, but the relevant box in the main window remained black for particular songs. 

 

Finally, the version 2.49 which I have been using up until now used to start playing where it left off when the program was last closed as soon as it started; I haven’t been able to find a setting to get the latest version to do this; I have to press Play to set it going. This isn’t really much of a problem for me.

 

If you would like me to send any of the settings or song files mentioned above, I can upload them or make them available for download for you.

 

P.S. During the above trials it occurred to me that it may be useful to have the Help > About box show the full paths to the program file and that to the settings file (cf Total Commander). This would enable the user to see which of the 32/64 bit versions and portable/installed versions is running.

 

Thank you once again for a program which has provided me with music for many years .

 

Regards from Dresden, Germany and Keep well, Chris van Zyl
